docs/development_suite/monitoring/introduction.md

```diff
@@ -19,3 +19,5 @@ The available Kubernetes resources are:
 - [**Cronjobs**](/development_suite/monitoring/resources/cronjobs.md):
 - [**Jobs**](/development_suite/monitoring/resources/jobs.md):
 - [**Namespaces**](/development_suite/monitoring/resources/namespace.md):
+
+From version 13.2.3, if you properly configured and deployed, you will also be able to monitor the status of one or more  one or more [**Custom Resources**](/console/design-your-projects/custom-resources/custom-resources.md) in the _Custom Resources_ section of the sidebar.
```

Comment:
```suggestion
From version 13.2.3, if you properly configured and deployed, you will also be able to monitor the status of one or more [**Custom Resources**](/console/design-your-projects/custom-resources/custom-resources.md) in the _Custom Resources_ section of the sidebar.
```

What do you mean by `if you properly configured and deployed`? You mean the Custom Resource in the Console?